### ISSUE TITLE
Flags for commands in powershell using DarkGray as foreground are unreadable

### ISSUE THREAD

**Issue by crumdev** [neilpa/cmd-colors-solarized#45]
I just installed the solarized theme doing the registry add first, then updated the links, then copied the Set-SolarizedDarkColorDefaults.ps1 to my profile folder, and then added required text to the end of my profile. Everything looked great after running Out-Colors but once I started using it all of the flags for powershell commands are defaulting to the DarkGray for font color. This is making it unreadable for me. I looked in the Set-SolarizedDarkColorDefaults.ps1 but I do not see that being used except for background colors.

![image](https://user-images.githubusercontent.com/5273788/70555752-a9988c00-1b4d-11ea-9a66-5a8661d118f5.png)

`PS C:\WINDOWS\system32> git tfs clone $tfs $/IntraWebForms/AmexReporting --branches=all --workspace="c:\repos"`

![image](https://user-images.githubusercontent.com/5273788/70555510-28d99000-1b4d-11ea-8e0e-770b5a927265.png)

**Comment by shaun-guth-allscripts** [comment]
Came here to see if this was anyone else's experience too. I am also unable to read any of the flags.

**Comment by l8nite** [comment]
Reposting on my non-work account: Came here to say that I am having this same behavior too. The flags are impossible to read.

See `git cherry -v master` screenshot:
![image](https://user-images.githubusercontent.com/1711530/74885580-c8c05080-532a-11ea-9b73-fd3aeee6f6af.png)

**Comment by crumdev** [comment]
@l8nite I ended up going through the install steps again and it seems to have corrected my issue. Granted I tried to uninstall it first and can't remember the steps I took there but I did not completely remove everything.

I followed the github page instructions for running Update-Link copied the Set-Solarized-***.ps1 to my profile folder then added the line to my Microsoft.PowerShell_profile.ps1 file.

![image](https://user-images.githubusercontent.com/5273788/74897003-b6163d80-5363-11ea-8433-d07e843d5b54.png)
